wave_utils/src/math.cpp

```diff
@@ -86,4 +86,12 @@ double wrapTo360(double euler_angle) {
     }
 }
 
+Mat3 skew_symmetric_matrix(Vec3 a){
+  Mat3 skew_matrix;
+  skew_matrix << 0, -a(2), a(1),
```

Comment:
Yeah clang-format is good, just this one little shortcoming. [This method](http://stackoverflow.com/a/34362422/431033) seems OK for a small matrix (`//` at the end of each line).